templates/cli/lib/commands/generic.js.twig

```diff
@@ -68,6 +69,13 @@ const client = new Command("client")
             try {
                 let url = new URL(endpoint);
                 if (url.protocol === "http:" || url.protocol === "https:") {
+                    let apClient = new Client().setEndpoint(endpoint);
```

Comment:
```suggestion
                    let client = new Client().setEndpoint(endpoint);
```
Let's be consistent with the other places we use the `Client()` class

Reply:
There is already a `client` defined in this scope.

Reply:
This will override the other `client` object.